### DISCUSSION TITLE
Goodbye Tine20

### DISCUSSION THREAD

**Discussion by rwkstgtd** [tine20/tine20#7380] [General]
I used the calendar-module since May 2013, quite some time now. I liked the interface, though it's not aging very well. I liked some functions such as the possibility to copy appointments from one date to another. I never liked that appointments in shared calendars are treated as if they were in one's personal calendar, without any possibility to filter those appointments.
I know it's free and I think pschuele is doing his best to keep the project running although not fulltime, probably. So criticism should not be too hard on the project. But: I'm leaving because of the bugs. It started with the interface needing minutes to load. I figured this one out myself: Full IMAP-Mailboxes are not good for Tine20. Most annoying fact: You can never save your credentials in the browser, even if you log into Tine20 like 10 times a day. Then those minor issues, like missing logo, hard to customise .htaccess, outdated docu, an update that keeps appointments from loading, updates that won't update and so on...
I migrated my calendar data to Nextcloud today, because for some months now it's impossible to update Tine20 (after you stopped releasing tine20-allinone_xxxxx.zip files). I use a shared hosting environment and the new format of releases is not suitable for me. It's hard (undocumented) to figure out what files are actually required. And when I just thought, I got it, the "vendor" folder was missing and it's nowhere to be found in the gigantic pile of data, each release is composed of.
So of course, I could make the next effort, do some research, post an issue. But it's way too much for a private use like mine (managing a family of 4). Nextcloud is set up in minutes, comes with automatic updates and the export from Tine20 took 5 times longer than the import into Nextcloud. With some tricky parts to figure out on the Tine20 side. That’s Tine20.
Fine groupware but way too complicated. I don't get why you made updating this complicated. Maybe you want people to move to the commercial version. Well, I move to Nextcloud. It's free and suitable for private use. Good luck with the project...

**Comment by lab-at-nohl** [comment]
I see your points (and frustration) but overall it comes down to the outdated documentation imho. Speed issues may be connected to limitations of shared hosting. 

To build yourself from source is a pain. Do not go there. Stupid question, have not tried: Do the [nightly builds](https://github.com/tine20/tine20/releases/)  released on a weekly basis not replace former releases? Specifically I mean the tar.gz file called nightly. What is missing, vendor et al. are not sufficient?

I think Tine and Next Cloud have similarities for Calendar. Still, Next Cloud comes from file sharing. Is there a official Mail app today? And finally, I prefer ActiveSync over xDAV, thus it is not a replacement. 

However, I think that communication to community is kind of broken. Because usually a non-professional do not follow Software on a weekly basis. We need kind of featured releases occasionally. 

Best
Johannes

**Comment by rwkstgtd** [comment]
Thanks for the reply, Johannes. Yep, you're right, the nightly builds would probably have done the job. I missread that part of the "release" section. I actually assumed, the tine20-folder of the "source"-zip-file would do the same job (.zip beeing better to handle on windows machines). I did not know the concept of build/source with php-code. There is nothing to build (=compile) with php, but of course you can package it differently, this I overlooked.
But there we go with the docu-issue again (yeah, you're right): I googled activesync-support (not ever having used it with Tine20), first hit you get is about broken active sync and which part of the supplied .htaccess you have to comment out. I remember spending some time on finding out the exact same thing on my broken Tine20 install: To comment out the "location"-part of supplied .htaccess. Lot's of people have to debug the same thing due to misssing docu. This is classic Tine20.
But yes, you're right, Nextcloud comes from filesharing. There is an official mail-client. I never tried it, I just need the calendar part of any groupware.

Cheers,
R.

**Comment by Taste-IT** [comment]
Hi,
i partly understand your frustration. I think the Dev, it seems that its only pschuele, do his best. If i could i would support, but i have no time at all. 

I use tine and Nextcloud. The mail app on NC is developed continuous, but its not good. Loading of my IMAP Folders sometimes work, after the last release it wont and its very slow. Actually i cant add any IMAP account, no connection to the mail server, but with other apps it work and i see no problems on serverside. I think its an problem with the mail app, again. The active sync in tine works very well and i have no performance issue on all IMAP accounts with all the folder i have. 
You are right the process of updating could be better and simpler, but it is what it is. If there is a new update, i first delete all files and than copy all files, except the config and .htaccess, from full package. I must say that i only use the calendar, address book and mail. I stopped changing the logo and login page, because its cost to much time. 
Sometimes i want go away from tine, because the support in the forum is bad and often there are no answers of problems. At all tine runs good with less problems and i can install it direct to my webspace. If i could include my nextcloud, owncloud or seafile filespace it would be perfect.

**Comment by agent-ck** [comment]
No, what the initial poster wrote is absolutely true. Since Tine20 switched completely to Github, it only went in one direction and that was down. My last installation attempt clearly shows where the problems are. 

It already starts with the installation. I found a total of 4 different installation guides, but not a single one worked, not even the Docker version! I ended up only being able to install Tine via tarball, but to get it to work I had to fall back on an existing Apache configuration from my employer. 

The setup was a horror! Connecting to the Active Directory didn't work at all, because Tine couldn't establish an encrypted connection, neither via SSL, nor via STARTTLS. Cause unknown, because even in loglevel 9 there was only the terse sentence, "Could not retrieve bind information". Worst of all, Setup kept getting the configuration wrong and I had to change it directly in the database to get the correct settings stored. 

Next drama was getting the intial admin logged in. It took me countless attempts and eventually it was done, but for the life of me I can't remember what I did to make it work.

Now as I was able to log in, I started the configuration of Tine20. At some point - still in the configuration - nothing was working anymore. After reloading, I could no longer log in. I had to completely reset the database and start over. At that point I had not backed up yet - everything was lost!

At work, we have meanwhile migrated from Tine20 to M365. I wanted to keep Tine20 going privately because I'm contibuting John to develop tbsync. But I can't work with such an unstable product and if I'm honest, I'm not willing to sacrifice days for an installation/update. My free time is more wisely invested in other things than Tine20.

I really hope you get back on track because Tine could actually be much better. But what you deliver at the moment is an insane joke.

I'm out as well!

**Reply by rwkstgtd** [reply]
Your post brings back bitter sweet memories I almost had forgotten. Install was rather bitter most of the time (exactly as you mentioned, getting stuck and so on), sweet the moment of success, when after endless retries it finally works - you'd never know why neither remember how you got there. I left that part out in the initial post, because this was ages ago, when I started using tine. Apparently there is still some fun in store for people doing a fresh install. ;-)
